src/bin/test/main.rs

```diff
@@ -63,20 +74,17 @@ fn update(board: Box<Board>, boards: Arc<Vec<BitBoard>>, game: &mut Game) -> (St
 
     //Uncomment for user input on White.
```

Comment:
Is the end goal to have both options available? AI vs. AI and Human vs AI? Or is the AI vs. AI available right now just for testing?

Reply:
The end goal would be to have the option of the ai playing against itself, the ai playing against a human, and the ai playing against a different ai.  I haven't gotten around to implementing a graceful way to switch between ai v ai and ai v human so for now I have just been commenting and uncommenting that code.

Reply:
Sounds good!